commit b90f6b0ed95589fee97460e31b5fb3eeb72dd856 (refs/changes/78/4978/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-10-19T09:16:11-07:00 (12 months ago)
    
    joyent/node-workflow#164 Add Buffer class and [set|clear]Immediate to wf-runner sandbox

diff --git a/lib/task-runner.js b/lib/task-runner.js
index cfb59ab..5b081ce 100644
--- a/lib/task-runner.js
+++ b/lib/task-runner.js
@@ -1,5 +1,5 @@
 // Copyright 2014 Pedro P. Candel <kusorbox@gmail.com>. All rights reserved.
-// Copyright (c) 2017, Joyent, Inc.
+// Copyright (c) 2018, Joyent, Inc.
 
 var bunyan = require('bunyan');
 var util = require('util');
@@ -53,8 +53,13 @@ var WorkflowTaskRunner = module.exports = function (opts) {
         console: global.console,
         util: global.util,
         info: info,
+        Buffer: global.Buffer,
         Error: Error
     };
+    if (global.setImmediate) {
+        sandbox.setImmediate = global.setImmediate;
+        sandbox.clearImmediate = global.clearImmediate;
+    }
 
     if (opts.sandbox) {
         if (typeof (opts.sandbox) !== 'object') {
diff --git a/package.json b/package.json
index 7010cf5..22550ee 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "wf",
     "description": "Tasks Workflows orchestration API and runners",
-    "version": "1.2.0",
+    "version": "1.2.1",
     "repository": {
         "type": "git",
         "url": "git://github.com/joyent/node-workflow.git"
